lib/facebook_ads/ad_objects/server_side/custom_data.rb

```diff
@@ -83,54 +87,46 @@ class CustomData
       # @param [Integer] num_items
       # @param [String] status
       # @param [String] search_string
-      def initialize(value: nil,
-                     currency: nil,
-                     content_name: nil,
-                     content_category: nil,
-                     content_ids: nil,
-                     contents: nil,
-                     content_type: nil,
-                     order_id: nil,
-                     predicted_ltv: nil,
-                     num_items: nil,
-                     status: nil,
-                     search_string: nil)
+      def initialize(**data)
```

Comment:
@minmin15 what would be the reason to remove the initialization params?

Reply:
@sheel I think we need to make changes as minor as possible, so instead of using `**data`, I decided to switch back to initialization params and adding `custom_properties`